src/app/ArticleEditing/page.tsx

```diff
@@ -0,0 +1,32 @@
+"use client"
+import React, { useState } from 'react';
+import ArticleEditor from '../_components/ArticleEditor';
+import html2md from 'html-to-md'
```

Comment:
if we're importing remark, we can use remark-html, which will also capture the underlined tags you have a replace statement for below

Reply:
Resolved. No longer using html2md package or remark for Article Editor as react-quill is no longer used for the rich text editor.